### PR TITLE
[BTS-160]: TextArea component

### PR DESCRIPTION
**Pull request by rapahaeru**
textarea

### CODE REVIEW

File: components/TextArea/TextArea.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  HELPER_TEXT_STYLE,
+  REQUIRED_MARK_STYLE,
+  ERROR_MESSAGE_STYLE,
+  AUTO_FILL_STYLE,
+} = INPUT_STYLE;
+
+const TextAreaTag = styled.textarea`
+  ${DEFAULT_INPUT_STYLE};
+
+  display: block;
```
Comment (allyssonsantos) [inline components/TextArea/TextArea.jsx]: I think the default display value of textarea element is already `block`.

File: components/TextArea/TextArea.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+  return (
+    <FieldGroup>
+      {label && (
+        <TextAreaLabel htmlFor={_id}>
+          {label}
+          {required && <RequiredMark>*</RequiredMark>}
+        </TextAreaLabel>
+      )}
+      <TextAreaTag error={error} id={_id} {...rest} />
```
Comment (allyssonsantos) [inline components/TextArea/TextArea.jsx]: Isn't valuable to pass the "required prop" to the TextAreaTag? for the browser show his default error message and for accessibility improvement?

Comment (ddsilva) [reply components/TextArea/TextArea.jsx]: Done

File: components/TextArea/TextArea.jsx (hunk 1 above)
Comment (ddsilva) [reply components/TextArea/TextArea.jsx]: Unfortunately, no :cry: 

Screenshot from 2019-03-22 08-57-06

Comment (allyssonsantos) [reply components/TextArea/TextArea.jsx]: :disappointed:

**Comment by rapahaeru** [comment]
Captura de tela de 2019-03-22 14-55-24

File: components/Input/Input.jsx [hunk 3]
```diff
@@ -117,14 +117,14 @@ class Input extends React.Component {
...
-  static counter = 0;
-
   constructor(props) {
     super(props);
 
     const { type } = props;
 
     this.state = { type };
+
+    this._id = props.id || uniqId('input-');
```
Comment (ggdaltoso) [inline components/Input/Input.jsx]: Destructuring?

File: stories/TextArea/TextArea.story.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { AutoExample } from '@catho/quantum-storybook-ui';
+
+import { IconFont } from '../../components/GlobalStyle';
+import TextArea from '../../components/TextArea';
+
+storiesOf('Forms', module).add('Text area', () => (
+  <>
+    <IconFont />
```
Comment (ggdaltoso) [inline stories/TextArea/TextArea.story.jsx]: Why not for all stories?

**Pull request merged**